examples/dgmulti_2d/elixir_advection_diffusion_nonperiodic.jl

```diff
@@ -0,0 +1,69 @@
+using Trixi, OrdinaryDiffEq
+
+dg = DGMulti(polydeg = 3, element_type = Quad(), approximation_type = Polynomial(),
+             surface_integral = SurfaceIntegralWeakForm(flux_lax_friedrichs),
+             volume_integral = VolumeIntegralWeakForm())
+
+get_diffusivity() = 5.0e-2
```

Comment:
Why is it named `get_diffusivity` and not just `diffusivity` (like we do for `density` or `pressure`)?

Reply:
No particular reason. We can use `diffusivity` for consistency (same with `get_Re()`, `get_Pr()`.

Reply:
👍